lib/ui/views/projects/project_details_view.dart

```diff
@@ -157,24 +157,26 @@ class _ProjectDetailsViewState extends State<ProjectDetailsView> {
 
   Widget _buildProjectDescription() {
     return Padding(
-      padding: const EdgeInsets.symmetric(vertical: 4.0),
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          Text(
-            'Description :',
-            style: Theme.of(context).textTheme.headline6.copyWith(
-                  fontWeight: FontWeight.bold,
-                  fontSize: 18,
-                ),
-          ),
-          Html(
-            data: """${_recievedProject.attributes.description ?? ''}""",
-            style: {'body': Style(fontSize: FontSize(18))},
-          )
-        ],
-      ),
-    );
+        padding: const EdgeInsets.symmetric(vertical: 4.0),
```

Comment:
Unnecessary padding without any text can be removed. Better to just return a `Container` when the condition is fulfilled so no unnecessary padding is there.